packages/config/README.md

```diff
@@ -15,7 +15,7 @@ npm install --save hops-config
 
 # Usage
 
-Please find a list of the default options below. They can be set in your project's package.json and can be overridden by using [npm config](https://docs.npmjs.com/cli/config) in the command line. You can extend hops-config within your project allowing you to use it to configure your isomorphic applications. The configuration is immutable at runtime - if you import it in your own application, you can only read its values, but you cannot alter them.
+Please find a list of the default options below. They can be set in your project's package.json. You can extend hops-config within your project allowing you to use it to configure your isomorphic applications.
```

Comment:
As this is the first mention of the usage, I'd probably write something like: "They can be set in your project's package.json (or dedicated files - see below)."